### ISSUE TITLE
Sand built on snow doesn't fall.

### ISSUE THREAD

**Issue by brachiel** [bravoserver/bravo#298]
When a sand block is built on top of a snowy block (a block with snow on top), the sand doesn't fall. Only after removing the snow, the sand block falls.

<a href="http://img822.imageshack.us/img822/1307/snowdoesntfall.png">Screenshot</a> (imageshack.us)

I'm using the following hooks (default bravo_example.ini):

dig_hooks = give, bravo_snow, alpha_sand_gravel, torch, water, lava, grass
pre_build_hooks = build_snow, tile, fertilizer
post_build_hooks = bravo_snow, alpha_sand_gravel
automatons = water, lava, trees, grass

**Comment by EntityReborn** [comment]
Is this a duplicate of https://github.com/MostAwesomeDude/bravo/issues/7 ?

**Comment by brachiel** [comment]
I would say no; since issue 7 is about placing a block on the snow-block itself. The issue described above is only related to blocks that fall (sand and gravel) - apparenlty they only fall through air; when they also should fall through snow.

**Comment by EntityReborn** [comment]
"fall through snow", thereby destroying said snow?
EDIT: Probably a silly question.

**Comment by MostAwesomeDude** [comment]
Seems like a curious interaction between build_snow and alpha_sand_gravel.

**Comment by MostAwesomeDude** [comment]
Marking this one for 1.9. I have a strong feeling that this is fixed, but I wanna be certain.

**Comment by MostAwesomeDude** [comment]
Yep, closing as fixed. Lemme know if you can still reproduce it; I can't.
